Containerfile

```diff
@@ -0,0 +1,58 @@
+FROM fedora:latest
```

Comment:
what is  this included for?

Reply:
Please ignore this file, currently im running integration tests in container locally and using this file.